### PR TITLE
Replace the Logo of Layer5 with Meshery in config.yml workflow

### LINKED ISSUE #1106: Replace the Logo of Layer5 with Meshery in ```config.yml``` workflow
### Description

Currently, we are using the Logo of Layer5 instead of Meshery. This repository is under the organization Meshery, so, there should be the image of Meshery.

**How to resolve this?**
- Go to https://github.com/meshery/meshery.io/tree/master/.github/welcome and replace this welcome image with the new one: 

image

Image Link: https://ibb.co/Wckh3qc

- Download this image, and upload with the correct name for future reference. The name of the image should be ```Meshery-celebration```.
- After uploading the image, and removing the previous image, go to https://github.com/meshery/meshery.io/blob/master/.github/config.yml and update the reference to previous image to the new image in the following part:

```
firstPRMergeComment: >
  Thanks for your contribution to the Layer5 and Meshery community! :tada:
   
  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Layer5-celebration.png)
  &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; :star: Please leave a [star on the project](../stargazers). :smile:
```
- That's it.

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### LINKED ISSUE #1116: Replace the logo of Layer5 with Meshery in config.yml workflow
**Description**

Currently, we are using the Logo of Layer5 instead of Meshery. This repository is under the organization Meshery, so, there should be an image of Meshery.

**How to resolve this**

- Go to .github and replace this welcome image with the new one:

- Make sure to display the images conditionally, based on the theme selected by user.
Here's the sample code:

```
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/GiorgosXou/Random-stuff/main/Programming/StackOverflow/Answers/70200610_11465149/w.png">
  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/GiorgosXou/Random-stuff/main/Programming/StackOverflow/Answers/70200610_11465149/b.png">
  <img alt="Meshery Logo" src="">
</picture>
```

New images are:

image

image

- Download these images, and upload with an appropriate name. Make sure to use conditional rendering as described above.

- After uploading the image, and removing the previous image, go to .github/config.yml folder and update the reference to previous image to the new image in the following part:

```
firstPRMergeComment: >
  Thanks for your contribution to the Layer5 and Meshery community! :tada:
   
  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Layer5-celebration.png)
  &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; :star: Please leave a [star on the project](../stargazers). :smile:
```
Here, replace `Congrats!` with the code

```
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/GiorgosXou/Random-stuff/main/Programming/StackOverflow/Answers/70200610_11465149/w.png">
  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/GiorgosXou/Random-stuff/main/Programming/StackOverflow/Answers/70200610_11465149/b.png">
  <img alt="Shows a black logo in light color mode and a white one in dark color mode." src="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
</picture>
```

make sure to change the image sources with the path to image in the folder where you upload it.

- That's it.

---
## Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)

The meshery.io website uses Jekyll and GitHub Pages. Site content is found under the [`master` branch](https://github.com/meshery/meshery.io/tree/master).
- 📚 See the [Contributing to Meshery.io Website](https://github.com/meshery/meshery.io#contributing-to-the-mesheryio-website) section of the readme.md and other [contributing instructions](https://docs.meshery.io/project/contributing), too.
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.meshery.io)

### PR DESCRIPTION
**Pull request by anshgoyalevil**
**Description**

This PR fixes #1106 and fixes #1116

In reference to conversation: https://layer5io.slack.com/archives/CLVT4TSG4/p1682039993764409?thread_ts=1682030772.512149&cid=CLVT4TSG4

**Notes for Reviewers**
This PR fixes an issue which is already closed. I have linked this issue to the same issue, since the scope of that issue coincides with the scope of this PR.

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: .github/config.yml
Comment (vishalvivekm) [inline .github/config.yml:1]: Thanks @anshgoyalevil .
LGTM 👌 // @leecalcote

**Review by vishalvivekm — APPROVED** [review]

File: .github/config.yml
Comment (ShivangShandilya) [reply .github/config.yml:1]: Yup LGTM, @anshgoyalevil just do the changes @vishalvivekm mentioned and I think we're good to go

Comment (anshgoyalevil) [reply .github/config.yml:1]: Done @ShivangShandilya 

Apologies for the delay since my exams are going on.

**Review by thisiskaransgit — APPROVED** [review]
Looks good to merge, thank you for your contribution :tada:

File: .github/config.yml [hunk 1]
```diff
@@ -17,7 +17,11 @@ newPRWelcomeComment: >
 firstPRMergeComment: >
   Thanks for your contribution to the Layer5 and Meshery community! :tada:
    
-  ![Congrats!](https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Meshery-celebration.png)
+  <picture>
+  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Meshery-celebration-Dark.svg">
+  <source media="(prefers-color-scheme: light)" srcset="https://raw.githubusercontent.com/meshery/meshery.io/master/.github/welcome/Meshery-celebration-Light.svg">
+  <img alt="Meshery Logo" src="">
```
Comment (leecalcote) [inline .github/config.yml:23]: @anshgoyalevil thanks! 

Are we sure that an empty image tag works?

Comment (anshgoyalevil) [reply .github/config.yml:23]: Yes sir, I have manually verified that.

Comment (leecalcote) [reply .github/config.yml:23]: Oh, good. Thank you much, @anshgoyalevil.

**Pull request merged**